src/ui/pages/common/PageStatusIcon.js

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { formattedText } from '@entando/utils';
+import { unescape } from 'lodash';
 
 import { PAGE_STATUS_DRAFT, PAGE_STATUS_PUBLISHED, PAGE_STATUS_UNPUBLISHED } from 'state/pages/const';
 
-const PageStatusIcon = ({ status, differsFromPublished, title }) => {
+const PageStatusIcon = ({ status, differsFromPublished }) => {
+  let iconTitle = formattedText(`pages.status.${status}`, `${status}`, {});
   const classNameAr = ['fa fa-circle PageStatusIcon', `PageStatusIcon--${status}`];
   if (differsFromPublished) {
     classNameAr.push('PageStatusIcon--diff');
+    iconTitle = [
+      `${formattedText('pages.status.draft')}`,
+      `${unescape('\u2260')}`,
```

Comment:
this function is deprecated. Using this decodeURI('\u2260').
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/unescape